src/nvim/eval/funcs.c

```diff
@@ -0,0 +1,106 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check
+// it. PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
+
+#include <stdio.h>
+#include <stddef.h>
+#include <nvim/eval.h>
+
+#ifdef INCLUDE_GENERATED_DECLARATIONS
+# include "funcs.generated.h"
```

Comment:
To include header for static function declarations use `#include "eval/funcs.c.generated.h"`.